pkgs/development/python-modules/mkdocs-material-insiders/default.nix

```diff
@@ -0,0 +1,103 @@
+{
+  # Evaluation
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pythonOlder,
+
+  # Build
+  hatchling,
+  hatch-requirements-txt,
+  hatch-nodejs-version,
+  trove-classifiers,
+
+  # Dependencies
+  jinja2,
+  markdown,
+  mkdocs,
+  mkdocs-material-extensions,
+  pygments,
+  pymdown-extensions,
+  babel,
+  colorama,
+  mergedeep,
+  paginate,
+  regex,
+  requests,
+
+  # Optional dependencies
+  mkdocs-minify-plugin,
+  mkdocs-redirects,
+  mkdocs-rss-plugin,
+  mkdocs-git-committers-plugin-2,
+  mkdocs-git-revision-date-localized-plugin,
+  pillow,
+  cairosvg,
+}:
+
+buildPythonPackage rec {
+  pname = "mkdocs-material";
+  version = "9.5.49-insiders-4.53.14";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "squidfunk";
+    repo = "mkdocs-material-insiders";
+    private = true;
```

Comment:
imo this package should be in your own repo, we already have [mkdocs-material](https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/development/python-modules/mkdocs-material/default.nix) in nixpkgs which can be built by hydra without impure trick